envoy/stream_info/stream_info.h

```diff
@@ -238,6 +238,26 @@ struct UpstreamTiming {
   absl::optional<MonotonicTime> last_upstream_rx_byte_received_;
 };
 
+// Measure the number of bytes sent and received for an http stream.
```

Comment:
```suggestion
// Measure the number of bytes sent and received for a stream.
```

(Since I bet we will implement this later for TCP)